ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -1300,6 +1348,39 @@ def reboot_dut(self):
 
         return
 
+    def restart_service(self):
+        for service_name in self.test_params['service_list']:
+            if 'image_path_on_dut' in self.service_data[service_name]:
+                stdout, stderr, return_code = self.dut_connection.execCommand("sudo sonic-installer upgrade-docker {} {} -y --warm".format(service_name, self.service_data[service_name]['image_path_on_dut']), timeout=30)
```

Comment:
I think `sonic-installer` won't work on 201911 as it still uses `sonic_installer`. Do you not plan to run these tests on 201911 images?

If yes, then in general it is good to check for image version.

Reply:
We won't support for 201911, I will add a check for the version.